recipes/stlab/config.yml

```diff
@@ -1,11 +1,3 @@
 versions:
-  "1.6.2":
-    folder: all
-  "1.5.6":
-    folder: all
-  "1.5.5":
-    folder: all
-  "1.5.4":
-    folder: all
-  "1.5.2":
+  "1.7.1":
```

Comment:
Missing old versions?

Reply:
I will add them after all the other things be solved.

Reply:
bump https://github.com/conan-io/conan-center-index/pull/13048/files#r1045138358

Reply:
Finally, they were dropped. Check https://github.com/conan-io/conan-center-index/pull/13048#issuecomment-1602618986